working/augmentation-libraries/feature-specification.md

```diff
@@ -932,8 +1082,13 @@ potentially non-redirecting property of the constructor.
 
 It is a compile-time error if:
 
-*   The augmented constructor has an initializer list or a body, or it has a
-    redirection.
+*   The augmented constructor has any initializers (including a redirecting
+    initializer).
```

Comment:
I don't think there is such a thing as a redirecting initializer. I think it could just be deleted:
```suggestion
*   The augmented constructor has any initializers.
```